index.js

```diff
@@ -11,8 +11,8 @@
    limitations under the License.
  */
 
-const { execSync } = require("child_process");
-const { Octokit } = require("@octokit/action");
+const {execSync} = require("child_process");
+const {Octokit} = require("@octokit/action");
```

Comment:
Can we please format the entire file using prettier?

Reply:
Done :) Wasn't sure what formatter was used